packages/rum-core/src/common/config-service.js

```diff
@@ -98,18 +104,17 @@ class Config {
 
       distributedTracing: true,
       distributedTracingOrigins: [],
-      distributedTracingHeaderValueCallback: undefined,
+      distributedTracingHeaderValueCallback: getDtHeaderValue,
```

Comment:
I think we might remove this config option once our we transition to the w3c `traceparent` header.